target/linux/ath79/dts/qca9531_comfast_cf-wr752ac.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-wr752ac", "qca,qca9531";
+	model = "COMFAST CF-WR752AC";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &lan;
+		led-failsafe = &lan;
+		led-upgrade = &lan;
```

Comment:
I'm not a fan of using lan and wlan for indication here.

Reply:
I don't really like it too, but as far as I can see, Comfast uses the LAN led too for signaling events. There are no other vacant leds available.